sanitize/utils.go

```diff
@@ -0,0 +1,47 @@
+package sanitize
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"strings"
+)
+
+func (s *Sanitizer) hashValue(value string) string {
+	hasher := sha256.New()
+	hasher.Write([]byte(s.salt + value))
+	return hex.EncodeToString(hasher.Sum(nil))
+}
+
+func (s *Sanitizer) sanitizeValue(value string) string {
+	hashedPath := s.hashValue(value)
+	if !strings.Contains(value, "/") {
+		return hashedPath
+	}
+
+	var redactedPath string
+
+	// this means it is a path field
```

Comment:
Why do we want special handling for path fields? Could there be any other fields like this which need special handling?

Reply:
Gateway expects `/` or `~/` in the beginning of paths. Similarly for content-types, it expects a `\` in between.
Thus, it needs special handling. Otherwise, the validation and sync would fail